hack/update-codegen.sh

```diff
@@ -40,6 +40,6 @@ CLIENTSET_PKG_NAME=clientset \
   github.com/grafana/grafana/pkg/generated \
   github.com/grafana/grafana/pkg/apis \
   github.com/grafana/grafana/pkg/apis \
-  "snapshots:v0alpha1" \
+  "datasource:v0alpha1" \
```

Comment:
Is this generic enough to keep in the main hack script?

Reply:
I think it is fine for now -- no specific value is better/worse and easier to keep the active one in the PR 🤷🏻